### PR TITLE
Better texts for Raster component

### PR DESCRIPTION
**Pull request by ger-benjamin**
For GSGMF-1502

That would be strange to have only the loading spinner in the button. I've put `Raster` instead. That's the same behaviors as the next button `Coordinate`. If no data is displayed (like out of CH), it displays `Coordinate`.

### CODE REVIEW

File: contribs/gmf/src/raster/widgetComponent.html [hunk 1]
```diff
@@ -16,12 +16,12 @@
       <span ng-show="ctrl.elevationLoading" class="fa custom-spinner-generic">
           <%=require('gmf/icons/spinner.svg?viewbox&height=1rem')%>
       </span>
-      <span ng-show="!ctrl.elevationValue && !ctrl.elevationLoading" translate>Elevation…</span>
+      <span ng-show="!ctrl.elevationValue && !ctrl.elevationLoading" translate>Raster</span>
```
Comment (sbrunner) [inline contribs/gmf/src/raster/widgetComponent.html:19]: => ?
```
      <span ng-show="ctrl.elevationLoading || !ctrl.elevationValue" class="fa custom-spinner-generic">
          <%=require('gmf/icons/spinner.svg?viewbox&height=1rem')%>
      </span>
```

Comment (ger-benjamin) [reply contribs/gmf/src/raster/widgetComponent.html:19]: I'll need words...

Comment (sbrunner) [reply contribs/gmf/src/raster/widgetComponent.html:19]: ?

Comment (ger-benjamin) [reply contribs/gmf/src/raster/widgetComponent.html:19]: I don't understand what you means with `=> ?`

Comment (ger-benjamin) [reply contribs/gmf/src/raster/widgetComponent.html:19]: If you mean "I've said that it's better to always have a loading spinner", I've replied in the first comment of the PR

Comment (sbrunner) [reply contribs/gmf/src/raster/widgetComponent.html:19]: OK I understand better =>
+ For Raster when the mouse is out of the map
But actually, it's strange to have when we move the mouse on the maps
'Elevation...' then loading icon then the value
For me, we should just have the loading icon then the value

Comment (ger-benjamin) [reply contribs/gmf/src/raster/widgetComponent.html:19]: Yes but it's strange to have the loading symbol forever when you are on tools panel or on the layertree... It looks like a bug.
I don't know... we need a third opinion @ybolognini @arnaud-morvan @RenataMuellerC2C ?

Comment (sbrunner) [reply contribs/gmf/src/raster/widgetComponent.html:19]: > Yes but it's strange to have the loading symbol forever when you are on tools panel or on the layertree... It looks like a bug.

On my last comment I don't say that... (+1 For Raster when the mouse is out of the map)

Comment (ger-benjamin) [reply contribs/gmf/src/raster/widgetComponent.html]: Ok, I'll display the 'loading spinner' always on the map when it has no value. And `Raster` when the mouse pointer is not on the map.

**Comment by ger-benjamin** [comment]
Done (+ some cleaning => move part of a big function into sub functions for clarity)

**Review by sbrunner — APPROVED** [review]
Excellent, thanks :-)

**Pull request merged**